### ISSUE TITLE
Conv_Rdtn and glue code in HydroDyn + FAST

### ISSUE THREAD

**Issue by enricogussoni** [OpenFAST/openfast#189]
Hi everyone,

I'm new in using HydroDyn and I'm trying to set up a new case but I got this error while running FAST:

```
...
Running HydroDyn (v2.05.01, 27-Jul-2016).
   Setting WaveTMax to 0.0 since WaveMod = 0

 FAST_InitializeAll:HydroDyn_Init:The value of Conv_Rdtn is not equal to the glue code timestep.
 This is not allowed in the current version of HydroDyn.

 FAST encountered an error during module initialization.
  Simulation error level: FATAL ERROR

  Aborting FAST.
```

Where do I modiphy the value of Conv_Rdtn?

Thank you

EG

**Comment by ghaymanNREL** [comment]
What version of FAST/OpenFAST are you running? I might be wrong, but I cannot locate your error message in the current OpenFAST codebase.  The following error check is in the current codebase:
"RdtnDT must be equal to the glue-code DT if PotMod = 1 and using RdtnMod > 0 or ExctnMod > 1."

**Comment by enricogussoni** [comment]
I'm using FASTv8.
Anyway I understand that the problem is in PotMod = 1.
PotMod = 1 can be used only with WAMIT, isn’t it?

**Comment by rafmudaf** [comment]
fixed by #314
